assets/js/job-submission.js

```diff
@@ -67,8 +67,96 @@ jQuery(document).ready(function($) {
 		return is_valid;
 	});
 
-	$( document.body ).on( 'submit', '.job-manager-form:not(.prevent-spinner-behavior)', function() {
-		$(this).find( '.spinner' ).addClass( 'is-active' );
-		$(this).find( 'input[type=submit]' ).addClass( 'disabled' ).on( 'click', function() { return false; } );
+	$( document.body ).on( 'submit', function() {
```

Comment:
I think the `.job-manager-form` selector still makes sense here. I'm not sure we want this to fire for all forms on a page.

Reply:
Right! Updated in 59cfd51

Reply:
Nice! Looks good and still works well for me. 🚢 🚢